corehq/pillows/mappings/app_mapping.py

```diff
@@ -2,7 +2,7 @@
 from corehq.pillows.mappings.utils import mapping_from_json
 from pillowtop.es_utils import ElasticsearchIndexInfo, APP_HQ_INDEX_NAME
 
-APP_INDEX = prefix_for_tests("hqapps_2020-02-26")
+APP_INDEX = prefix_for_tests("hqapps_2020-10-13")
```

Comment:
I believe this should also be updated [here](https://github.com/dimagi/commcare-hq/blob/master/testapps/test_elasticsearch/tests/test_prod_es_indices.py) and [here](https://github.com/dimagi/commcare-hq/blob/master/testapps/test_pillowtop/tests/data/all-pillow-meta.json).